src/observable.cpp

```diff
@@ -95,5 +142,3 @@ template class ConfigVar<double>;
 template class ConfigVar<int>;
 template class ConfigVar<std::string>;
 template class ConfigVar<wxString>;
```

Comment:
I don't think these are achieving anything?

Reply:
They limit the types the template can be instantiated with: https://stackoverflow.com/questions/2351148